lib/taiko.js

```diff
@@ -306,18 +307,20 @@ const _closeBrowser = async () => {
     // remove listeners other than JS dialog for beforeUnload on client first to stop executing them when closing
     await _client.removeAllListeners();
     pageHandler.addJavascriptDialogOpeningListener();
-    if (!defaultConfig.firefox) {
+    //TODO: Remove check once fixed https://bugs.chromium.org/p/chromium/issues/detail?id=1147809
+    if (!defaultConfig.firefox && !(os.type().includes('Windows') && defaultConfig.headful)) {
```

Comment:
I think this without having to import a library and having an exact match

https://nodejs.org/api/process.html#process_process_platform

```suggestion
    if (!defaultConfig.firefox && !(process.platform == 'win32' && defaultConfig.headful)) {
```

It also returns win32 for 64 bit systems. The same is available in `os` too.

https://nodejs.org/api/os.html#os_os_platform